src/main/kotlin/uk/gov/gdx/datashare/models/AcquirerSubscriptionDto.kt

```diff
@@ -1,16 +1,38 @@
 package uk.gov.gdx.datashare.models
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.gdx.datashare.enums.EnrichmentField
 import uk.gov.gdx.datashare.enums.EventType
 import java.time.LocalDateTime
 import java.util.*
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Do we necessarily want this?

Reply:
We do in all other cases, if there is a good reason to return nulls we can remove